Packs/GCP-IAM/Integrations/GCPIAM/GCPIAM.py

```diff
@@ -1239,13 +1239,21 @@ def generate_iam_policy_command_output(response: dict, resource_name: str = None
         readable_header = f'{resource_name} IAM policy information:'
     outputs = copy.deepcopy(response)
     outputs['name'] = resource_name
+    bindings = outputs.get("bindings", [])
+    if roles and bindings:
+        bindings_roles_only = []
+        for index, entry in enumerate(bindings):
+            if entry["role"] in roles:
+                bindings_roles_only.append(bindings.pop(index))
+
+        bindings = bindings_roles_only
```

Comment:
If you don't have any results after filtering, what happens?

Reply:
<img width="1710" alt="Screenshot 2023-10-31 at 12 44 54 AM" src="https://github.com/demisto/content/assets/40349459/e021f3cb-ebc6-4d2e-b3a5-ef56bc8e9fe4">

Reply:
<img width="1322" alt="Screenshot 2023-10-31 at 2 06 41 AM" src="https://github.com/demisto/content/assets/40349459/246bee0d-843d-48b6-be01-7ea1113a1a11">

Reply:
<img width="1322" alt="Screenshot 2023-10-31 at 2 06 41 AM" src="https://github.com/demisto/content/assets/40349459/3b20277e-56c7-433e-9258-7a884506d20f">